Description: Relax version constraint of the loom dependency
Author: Antonin Delpeuch <antonin@delpeuch.eu>
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -145,7 +145,7 @@ features = [
 version = "0.5.1"
 
 [target."cfg(oneshot_loom)".dependencies.loom]
-version = "0.7.2"
+version = ">= 0.5, < 0.8"
 features = ["futures"]
 optional = true
 
